configure.ac

```diff
@@ -1192,15 +1192,14 @@ if test "X$HDF5_DOXYGEN" = "Xyes"; then
   AC_SUBST([DOXYGEN_SERVER_BASED_SEARCH])
   AC_SUBST([DOXYGEN_EXTERNAL_SEARCH])
   AC_SUBST([DOXYGEN_SEARCHENGINE_URL])
-  AC_SUBST([DOXYGEN_STRIP_FROM_PATH])
```

Comment:
CMake change too?

Reply:
Will do.

Reply:
I think you fixed that already in #994 .

Reply:
Restored.